lib/Data/Xls.hs

```diff
@@ -129,13 +129,15 @@ CCALL(xls_cell_hidden,          XLSCell -> IO Int8 -- Int8)
 data XlsException =
       XlsFileNotFound String
     | XlsParseError String
+    | Libxls XLSError
```

Comment:
The two cases of XLSError are already represented in XlsException. We can add the other cases as well, something like this:

```
data XlsException =
      XlsFileNotFound String
    | XlsParseError String
    | XlsSeekError String
    | XlsReadError String 
    | XlsMallocError String
```

And just translate XLSError to XlsException, and then do not export XLSError.